### DISCUSSION TITLE
HTTP Request Timeouts on Dashboard

### DISCUSSION THREAD

**Discussion by arvai** [plausible/analytics#1908] [Self-Hosted Support] [answered]
Hi !

I have a Plausible setup with a quite large amount of data, currently it is running on a simple VPS (1CPU , 2GB RAM). The Dashboard requests (/main-graph, /sources, etc.) are failing if I want to get 30 days of aggregated data. I get this HTTP 500 error response:

"(exit) exited in: Task.await(%Task{owner: #PID<0.17751.1>, pid: #PID<0.17757.1>, ref: #Reference<0.1547955430.4120117249.73479>}, 10000)\n ** (EXIT) time out"

How could I eliminate these HTTP timeouts ? Set more CPUs for the VPS? Or Change the timeout somewhere ?

Thanks in advance,

**Comment by ukutaht** [comment, marked as answer]
Yes, you probably need more CPU and RAM.
